protostar/starknet/cache.py

```diff
@@ -0,0 +1,32 @@
+import os
+from os import path
+from typing import Optional
+import json
+
+
+class CacheIO:
```

Comment:
Could you please submit a PR which writes a `.gitignore` file containing `*` rule in cache directory? We don't want people to commit this dir into their repositories.

Reply:
Hmm, I'm not sure about this honestly, are you? I don't think that popular tools do such things. Let's say you do something like this
```
git init
yarn init
yarn add express
```
You have to handle the `.gitignore` yourself, `yarn` will not do that for you.
It's just a simple example, we can discuss this further and I may not be right.
Please, note that a user will have to add the generated `.gitignore` to their repository anyway. For me it just makes more sense for me to add some rules for `*cache` (or something like that) to their `.gitignore` file.
At the end of the day, they may not be using git, there may be other tools like svn or something, right?

Reply:
Elixir tooling or IntelliJ IDEA do this for example.

![image](https://user-images.githubusercontent.com/3450050/194874354-ea8960e4-14dc-46bd-8344-d81018ab13a0.png)

This trick makes the whole `.elixir_ls` directory **immediately invisible** to Git, without having to commit anything.

Reply:
Okay, if you insist then I'm okay with adding this: https://github.com/software-mansion/protostar/pull/973